Assignment_one.py

```diff
@@ -0,0 +1,26 @@
+import requests
+import json
+import pytest
+
+url = 'https://automationexercise.com/api/productsList'
+
+def test_products_list():
+    response = requests.get(url)
+
+    # Validate status code
+    assert response.status_code == 200, "API is not returning a 200 status code"
```

Comment:
Please use framework approach

Reply:
@amitsahu8  Changes made kindly verify it

Reply:
I don't see any change

Reply:
@amitsahu8  Wrongly updated, Now I have updated correctly. Kindly verify it